tests/plugins/balloontoolbar/context/focusablebuttons.js

```diff
@@ -0,0 +1,87 @@
+/* bender-tags: balloontoolbar */
+/* bender-ckeditor-plugins: balloontoolbar, basicstyles, button, toolbar */
+
+( function() {
+	'use strict';
+
+	bender.editor = {
+		startupData: '<p>Hello <strong>world</strong>.</p>',
+		config: {
+			extraAllowedContent: 'strong em'
+		}
+	};
+
+	bender.test( {
+		'test check if balloon toolbar elements are registered as focusable': function() {
+			var editor = this.editor,
+				toolbar = new CKEDITOR.ui.balloonToolbar( editor ),
+				before,
+				afterAdding,
```

Comment:
Why adding extra variables like that instead asserting right after given action happened? E.g. instead of putting it into `afterAdding` variable it should be simply checked right after attaching the toolbar.